.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,3 +18,20 @@ If you haven't already, read the full [contribution guide](../CONTRIBUTING.md).
 * [ ] I have opened a new issue for the proposal, or commented on an existing one, and ensured that the Bicep maintainers are good with the design of the feature being implemented
 * [ ] I have included "Fixes #{issue_number}" in the PR description, so GitHub can link to the issue and close it when the PR is merged
 * [ ] I have appropriate test coverage of my new feature
+
+## Contributing a snippet
+
+* [ ] I have only a single resource in my snippet
```

Comment:
I am not clear on this one. The remaining snippets that I am going to port from arm tools have dependsOn. So we are going to have more than one resource. 

@alex-frankel shared this during the DSL meeting- https://github.com/Azure/bicep/blob/main/docs/spec/resource-scopes.md#parent-child-syntax

So my understanding is we'll still have more than one resource in the template file. Let me know if I am missing something.

Reply:
For now, we'll stick to single resources. Quick Starts are a better home for multi-resource examples. Any ARM Tools snippets that more than one resource shouldn't be ported over.

I added multi-resource snippets to https://github.com/Azure/bicep/issues/2247